scripts/check_frontend_test_coverage.py

```diff
@@ -106,7 +106,8 @@
     'truncate-input-based-on-interaction-answer-type.filter.ts',
     'truncate.filter.ts',
     'tutor-card.directive.ts',
-    'unit-test-utils.ajs.ts', # Please don't try to cover this file.
+    # Please don't try to cover this file.
```

Comment:
This change is done because I introduced a new lint check to prevent trailing/inline comments unless the reason is a valid allowed pragma.

Reply:
Could you replace "this" in the comment with the file name? Thanks

Reply:
Does it look better now?

Reply:
Done.